2 Niagara Fundamentals Exercise - Multiplexer/multiplexer-rt/src/com/tridiumuniversity/multiplexer/BMultiplexer.java

```diff
@@ -0,0 +1,241 @@
+   * @see #in1
+   */
+  public void setIn1(boolean v) { setBoolean(in1, v, null); }
+
+  //endregion Property "in1"
+
+  //region Property "in2"
+
+  /**
+   * Slot for the {@code in2} property.
+   * @see #getIn2
+   * @see #setIn2
+   */
+  public static final Property in2 = newProperty(Flags.SUMMARY, false, null);
+
+  /**
+   * Get the {@code in2} property.
+   * @see #in2
+   */
+  public boolean getIn2() { return getBoolean(in2); }
+
+  /**
+   * Set the {@code in2} property.
+   * @see #in2
+   */
+  public void setIn2(boolean v) { setBoolean(in2, v, null); }
+
+  //endregion Property "in2"
+
+  //region Property "in3"
+
+  /**
+   * Slot for the {@code in3} property.
+   * @see #getIn3
+   * @see #setIn3
+   */
+  public static final Property in3 = newProperty(Flags.SUMMARY, false, null);
+
+  /**
+   * Get the {@code in3} property.
+   * @see #in3
+   */
+  public boolean getIn3() { return getBoolean(in3); }
+
+  /**
+   * Set the {@code in3} property.
+   * @see #in3
+   */
+  public void setIn3(boolean v) { setBoolean(in3, v, null); }
+
+  //endregion Property "in3"
+
+  //region Property "s0"
+
+  /**
+   * Slot for the {@code s0} property.
+   * @see #getS0
+   * @see #setS0
+   */
+  public static final Property s0 = newProperty(Flags.SUMMARY, false, null);
+
+  /**
+   * Get the {@code s0} property.
+   * @see #s0
+   */
+  public boolean getS0() { return getBoolean(s0); }
+
+  /**
+   * Set the {@code s0} property.
+   * @see #s0
+   */
+  public void setS0(boolean v) { setBoolean(s0, v, null); }
+
+  //endregion Property "s0"
+
+  //region Property "s1"
+
+  /**
+   * Slot for the {@code s1} property.
+   * @see #getS1
+   * @see #setS1
+   */
+  public static final Property s1 = newProperty(Flags.SUMMARY, false, null);
+
+  /**
+   * Get the {@code s1} property.
+   * @see #s1
+   */
+  public boolean getS1() { return getBoolean(s1); }
+
+  /**
+   * Set the {@code s1} property.
+   * @see #s1
+   */
+  public void setS1(boolean v) { setBoolean(s1, v, null); }
+
+  //endregion Property "s1"
+
+  //region Property "out"
+
+  /**
+   * Slot for the {@code out} property.
+   * @see #getOut
+   * @see #setOut
+   */
+  public static final Property out = newProperty(Flags.READONLY | Flags.SUMMARY, false, null);
+
+  /**
+   * Get the {@code out} property.
+   * @see #out
+   */
+  public boolean getOut() { return getBoolean(out); }
+
+  /**
+   * Set the {@code out} property.
+   * @see #out
+   */
+  public void setOut(boolean v) { setBoolean(out, v, null); }
+
+  //endregion Property "out"
+
+  //region Type
+
+  @Override
+  public Type getType() { return TYPE; }
+  public static final Type TYPE = Sys.loadType(BMultiplexer.class);
+
+  //endregion Type
+
+//@formatter:on
+//endregion /*+ ------------ END BAJA AUTO GENERATED CODE -------------- +*/
+
+  @Override
+    public void changed(Property p, Context cx)
+    {
+        if(!getS0() && !getS1()){
+            setOut(getIn0());
+            System.out.println("In0");
```

Comment:
This is fine for these exercises, but of course in production code you should make sure not to leave in any `System.out.println` calls.